merkle-tree/src/utils.js

```diff
@@ -117,6 +84,51 @@ function multiplyByNumber(num, x, base) {
   return result;
 }
 
+/**
+utility function to check that a string has a leading 0x (which the Solidity
+compiler uses to check for a hex string).  It adds it if it's not present. If
+it is present then it returns the string unaltered
+*/
+function ensure0x(hex = '') {
```

Comment:
Interesting - looks like we still need to import the zkp-utils library into timber :)

Reply:
yes, I do the change with new PR

Reply:
done, PR updated